libnavui-androidauto/src/main/java/com/mapbox/androidauto/screenmanager/MapboxScreenManager.kt

```diff
@@ -21,8 +23,57 @@ import java.util.ArrayDeque
 import java.util.Deque
 
 /**
- * The Mapbox Navigation Android Auto SDK is prepared with a default experience. This object allows
- * you to customize the experience to meet your needs.
+ * The Mapbox Navigation Android Auto SDK is prepared with a default and customizable experience.
+ * [MapboxScreenManager] allows you to prepare an experience that includes many screens with a
+ * single function. Use the [MapboxCarContext.prepareScreens] to get started.
+ *
+ * ### How it works
+ * Using [String] to [MapboxScreenFactory] pairing, the [MapboxScreenManager] will create a screen
+ * based on a string provided. You can change which screen should show by updating the key value pair.
+ *
+ * Communication with [MapboxScreenManager] can be done statically. Observe the screen changes
+ * with [MapboxScreenManager.screenEvent], and change the car screen with different
+ * [MapboxScreenOperation]. Each operation has requirements built into [MapboxScreenManager]. For
```

Comment:
```suggestion
 * Changes to screen can be observed using [MapboxScreenManager.screenEvent] whereas changes to car screen can be applied using [MapboxScreenOperation].  Each operation has requirements built into [MapboxScreenManager]. To invoke requirements from [MapboxScreenManager], you would need to create an instance of [MapboxScreenManager] in some cases whereas it can be referenced statically in others. For
```

Reply:
Yeah went through and reworded this